tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -153,6 +194,15 @@ install_cc_qemu() {
 	info "build static CC qemu"
 	export qemu_repo="$(yq r $versions_yaml assets.hypervisor.qemu.url)"
 	export qemu_version="$(yq r $versions_yaml assets.hypervisor.qemu.version)"
+
+	install_cached_component \
+		"QEMU" \
+		"${jenkins_url}/job/kata-containers-2.0-qemu-cc-$(uname -m)/${cached_artifacts_path}" \
```

Comment:
@fidencio to test this PR could you please change the name of the jenkins job as it is already existing one job with this name

Reply:
the same for the kernel and clh please